tests/conftest.py

```diff
@@ -3,10 +3,15 @@
 
 def pytest_addoption(parser):
     parser.addoption(
-        "--record", action="store_true", help="record new server-responses"
+        "--vcrmode",
+        action="store",
+        default="replay",
+        help="mode for vcr recording; one of ['replay', 'record', 'live']",
```

Comment:
I think you can use `choices` here like in argparse (https://docs.python.org/3/library/argparse.html#choices, https://docs.pytest.org/en/latest/reference.html#pytest.config.argparsing.Parser.addoption), then you don't need the `assert` later.

Reply:
Yes, I had to read `pytest` source to figure that out.

Reply:
Now that i found the docs, i know that this parser is _from_ argparse. ;)